feature_extractor/entity_helper.py

```diff
@@ -61,18 +59,19 @@ def __init__(self, config: nmmo.config.Config,
                        ModelArchitecture.NEARBY_NUM_FEATURES
 
   def reset(self, init_obs: Dict) -> None:
+    self.attack_target = [None] * self.team_size
     self._choose_professions()
     self.update(init_obs)
 
   # merge all the member observations and denormalize them into
   # self._entities, self._member_location, self._entity_features
   def update(self, obs: Dict) -> None:
-    self._entities = {} # id -> entity_ob
+    self._entities = {} # id -> entity_ob (1 row)
     self.member_location = {} # id -> (row, col)
     self._entity_features = {} # id -> entity_features
 
     for agent_id, agent_obs in obs.items():
-      entity_rows = agent_obs['Entity'][:,EntityAttr["id"]] != 0
+      entity_rows = agent_obs['Entity'][:, EntityAttr["id"]] != 0
```

Comment:
prefer no space after , in [

Reply:
removed the spaces in the cases of [:,EntityAttr], [:,ItemAttr]